labgrid/driver/resetdriver.py

```diff
@@ -24,3 +27,61 @@ def reset(self):
         self.output.set(True)
         time.sleep(self.delay)
         self.output.set(False)
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class BCUResetDriver(Driver, ResetProtocol, BootCfgProtocol):
```

Comment:
As BCU supports more than reset, perhaps this should be `BCUDriver` or `NXPBCUDriver`. And be in a separate file. Functions for power measurement could then be added to the driver later.

Reply:
Agree, we just met recently a situation where another function from bcu (set_boot_mode) was needed.